text-generation-inference/server/text_generation_server/models/causal_lm.py

```diff
@@ -348,10 +362,12 @@ def concatenate(cls, batches: List["CausalLMBatch"]) -> "CausalLMBatch":
             # BLOOM Keys:   [batch_size * num_heads, head_dim, seq_length]
             # BLOOM Values: [batch_size * num_heads, seq_length, head_dim]
             # And ensure that we can update tensors in-place
+            kv_tuple = False
```

Comment:
Same

Reply:
in https://github.com/huggingface/optimum-habana/pull/389/files#diff-244871853752a7a019853c852803a797a9a14dddb3d9d859ecbae4b316ed6974R457